server/utilities/audio/koala-filter.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "KoalaFilter"
+description: "Audio noise reduction filter using Koala AI technology from Picovoice"
+---
+
+## Overview
+
+`KoalaFilter` is an audio processor that reduces background noise in real-time audio streams using Koala Noise Suppression technology from Picovoice. It inherits from `BaseAudioFilter` and processes audio frames to improve audio quality by removing unwanted noise.
+
+To use Koala, you need a Picovoice access key. Get started at [Picovoice Console](https://console.picovoice.ai/signup).
+
+## Installation
+
+The Koala filter requires additional dependencies:
+
+```bash
+pip install pipecat-ai[koala]
+```
+
+You'll also need to set up your Koala access key as an environment variable: `KOALA_ACCESS_KEY`
+
+## Constructor Parameters
+
+<ParamField path="access_key" type="str" required>
+  Picovoice access key for using the Koala noise suppression service
+</ParamField>
+
+## Usage Example
+
+```python
+from pipecat.audio.filters.koala_filter import KoalaFilter
+
+transport = DailyTransport(
+    room_url,
+    token,
+    "Respond bot",
+    DailyParams(
+        audio_in_filter=KoalaFilter(access_key=os.getenv("KOALA_ACCESS_KEY")), # Enable Koala noise reduction
+        audio_out_enabled=True,
+        vad_enabled=True,
+        vad_analyzer=SileroVADAnalyzer(),
+    ),
+)
+```
+
+## Frame Flow
```

Comment:
Not sure if this is considered Frame flow.

Btw, it does support the `FilterEnableFrame` and the settings one, but because the transport passes it to them (i.e. they don't receive them from the pipeline directly, but indirectly through the transport).